examples/quickstart/src/main/java/quickstart/ReadmeSnippets.java

```diff
@@ -23,22 +23,22 @@
 import com.okta.sdk.resource.ExtensibleResource;
 import com.okta.sdk.resource.application.Application;
 import com.okta.sdk.resource.application.ApplicationList;
-import com.okta.sdk.resource.application.SwaApplication;
-import com.okta.sdk.resource.application.SwaApplicationSettings;
-import com.okta.sdk.resource.application.SwaApplicationSettingsApplication;
+import com.okta.sdk.resource.SwaApplication;
+import com.okta.sdk.resource.SwaApplicationSettings;
+import com.okta.sdk.resource.SwaApplicationSettingsApplication;
 import com.okta.sdk.resource.group.Group;
 import com.okta.sdk.resource.group.GroupBuilder;
 import com.okta.sdk.resource.group.GroupList;
-import com.okta.sdk.resource.log.LogEventList;
-import com.okta.sdk.resource.user.User;
+import com.okta.sdk.resource.system.log.LogEventList;
+import com.okta.sdk.resource.group.User;
 import com.okta.sdk.resource.user.UserBuilder;
-import com.okta.sdk.resource.user.UserList;
```

Comment:
I'm noticing a lot of the resource locations being changed to worse or even incorrect packages, e.g. `User` went from `user` to `group`. A few also ended up in the root resource directory. Two options here:

Having good grouping of resources seems nice to have. If this is the case, we probably should improve my super hacky attempt to backfill `model.getExtensions().get("x-okta-tags");`

Otherwise, we can just put EVERYTHING in the root, which is real messy but can simplify all the imports to `import com.okta.sdk.resource.*`, which is the vast majority of all of these PRs.

Reply:
Agreed, will need to package `User` object into `user` package.

Reply:
Refactored codegen to place all resources in root resource package.